manifests/ers/ers-dbwriter.yaml

```diff
@@ -14,8 +14,8 @@ spec:
         app: erskafka
     spec:
       containers:
-      - image: jmcarcell/ers-dbwriter:latest
+      - image: plaso/ers-db-writer
```

Comment:
Is this a "private" image?

Reply:
This was a temporary placeholder whilst we were waiting to get the image into the dunedaq docker repo. Will be fixed shortly

Reply:
New commit:
`spec:
      containers:
              - image: ghcr.io/dune-daq/pocket-ersdbwriter:latest`